r_markdown/background_online_analysis_ch4.Rmd

```diff
@@ -156,7 +156,8 @@ Calculate mean background over all good tests
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
   mean_bg <- dplyr::summarise(background_means,
                               ppm = mean(ppm, na.rm = TRUE),
-                              conc = mean(conc, na.rm = TRUE))
+                              conc = mean(conc, na.rm = TRUE),
```

Comment:
I assume the number is NA low, either the analyzer was logging or it wasn't. NA's prob occur when timestamps don't quite line up. Most NA probably isn't a problem for the pre-background, when you look at the plots we see a near constant value. The post-background where we see a slope is my main worry.